docs/reference/env-variables.rst

```diff
@@ -0,0 +1,703 @@
+      - uint
+      - 2048
+      - Set a limit in Mb on the maximum USWC allocation size, -1 = No limit
+    *
+      - AMD_SERIALIZE_KERNEL
+      - uint
+      - 0
+      - Serialize kernel enqueue, 0x1 = Wait for completion before enqueue, 0x2 = Wait for completion after enqueue 0x3 = both
+    *
+      - AMD_SERIALIZE_COPY
+      - uint
+      - 0
+      - Serialize copies, 0x1 = Wait for completion before enqueue, 0x2 = Wait for completion after enqueue 0x3 = both
+    *
+      - HIP_LAUNCH_BLOCKING
+      - uint
+      - 0
+      - Serialize kernel enqueue 0x1 = Wait for completion after enqueue, same as AMD_SERIALIZE_KERNEL=2
+    *
+      - PAL_ALWAYS_RESIDENT
+      - bool
+      - FALSE
+      - Force memory resources to become resident at allocation time
+    *
+      - HIP_HOST_COHERENT
+      - uint
+      - 0
+      - Coherent memory in hipHostMalloc
+    *
+      - AMD_OPT_FLUSH
+      - uint
+      - 1
+      - Kernel flush option , 0x0 = Use system-scope fence operations. 0x1 = Use device-scope fence operations when possible.
+    *
+      - AMD_DIRECT_DISPATCH
+      - bool
+      - FALSE
+      - Enable direct kernel dispatch.
+    *
+      - HIP_HIDDEN_FREE_MEM
+      - uint
+      - 0
+      - Reserve free mem reporting in Mb, 0 = Disable
+    *
+      - GPU_FORCE_BLIT_COPY_SIZE
+      - size_t
+      - 0
+      - Size in KB of the threshold below which to force blit instead for sdma
+    *
+      - ROC_ACTIVE_WAIT_TIMEOUT
+      - uint
+      - 0
+      - Forces active wait of GPU interrup for the timeout, us unit
+    *
+      - ROC_ENABLE_LARGE_BAR
+      - bool
+      - TRUE
+      - Enable Large Bar if supported by the device
+    *
+      - ROC_CPU_WAIT_FOR_SIGNAL
+      - bool
+      - TRUE
+      - Enable CPU wait for dependent HSA signals.
+    *
+      - ROC_SYSTEM_SCOPE_SIGNAL
+      - bool
+      - TRUE
+      - Enable system scope for signals, uses interrupts.
+    *
+      - GPU_FORCE_QUEUE_PROFILING
+      - bool
+      - FALSE
+      - Force command queue profiling by default
+    *
+      - HIP_MEM_POOL_SUPPORT
+      - bool
+      - FALSE
+      - Enables memory pool support in HIP
+    *
+      - HIP_MEM_POOL_USE_VM
+      - bool
+      - IS_WINDOWS
+      - Enables memory pool support in HIP
+    *
+      - PAL_HIP_IPC_FLAG
+      - bool
+      - FALSE
+      - Enable interprocess flag for device allocation in PAL HIP
+    *
+      - PAL_FORCE_ASIC_REVISION
+      - uint
+      - 0
+      - Force a specific asic revision for all devices
+    *
+      - PAL_EMBED_KERNEL_MD
+      - bool
+      - FALSE
+      - Enables writing kernel metadata into command buffers.
+    *
+      - ROC_GLOBAL_CU_MASK
+      - cstring
+      - 
+      - Sets a global CU mask , entered as hex value for all queues, Each active bit represents using one CU , e.g., 0xf enables only 4 CUs
+    *
+      - AMD_LOG_LEVEL_FILE
+      - cstring
+      - 
+      - Set output file for AMD_LOG_LEVEL, Default is stderr
```

Comment:
```suggestion
      - 
      - Set output file for AMD_LOG_LEVEL. Default is stderr
```
Also shouldn't the default be mentioned in the column for default values? Unless this is just the default behaviour, but not the default value for the environment variable